keeper/msg_server.go

```diff
@@ -480,7 +480,7 @@ func (k *Keeper) handleVaultsYieldSeasonTwo(ctx context.Context) error {
 	}
 
 	// Send the Staked vault yield to the Collector address.
-	err = k.bank.SendCoins(ctx, vaults.StakedVaultAddress, k.vaultsSeasonTwoYieldCollector, sdk.NewCoins(sdk.NewCoin(k.denom, yield)))
+	err = k.bank.SendCoins(ctx, vaults.StakedVaultAddress, k.GetVaultsSeasonTwoYieldCollector(ctx), sdk.NewCoins(sdk.NewCoin(k.denom, yield)))
```

Comment:
If we did check the error we are currently ignoring in `GetVaultsSeasonTwoYieldCollector` then this call would need to happen outside of the call to `SendCoins` so we can check the error that is returned